package.json

```diff
@@ -34,38 +34,36 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "branch-diff": "^1.10.5",
-    "chalk": "^5.0.1",
-    "changelog-maker": "^2.8.0",
+    "branch-diff": "^2.1.0",
+    "chalk": "^5.2.0",
+    "changelog-maker": "^3.2.1",
     "cheerio": "^1.0.0-rc.10",
     "clipboardy": "^3.0.0",
-    "core-validate-commit": "^3.16.0",
+    "core-validate-commit": "^3.18.0",
     "enquirer": "^2.3.6",
-    "execa": "^6.1.0",
-    "figures": "^4.0.1",
+    "execa": "^7.0.0",
+    "figures": "^5.0.0",
     "form-data": "^4.0.0",
```

Comment:
note for a future PR: Undici has its own `FormData` implementation, we should probably migrate to that.